src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,79 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import nextstep.subway.dto.LineRequest;
+
+@Entity
+public class Line extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    private String name;
+
+    @Column(unique = true)
+    private String color;
+
+    private Long distance;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id")
+    private Station upStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id")
+    private Station downStation;
+
+    protected Line() {
+
+    }
+
+    public Line(String name, String color, Long distance, Station upStation, Station downStation) {
+        this.name = name;
+        this.color = color;
+        this.distance = distance;
+        this.upStation = upStation;
+        this.downStation = downStation;
+    }
+
+    public static Line of(LineRequest lineRequest, Station upStation, Station downStation) {
+        return new Line(lineRequest.getName(), lineRequest.getColor(), lineRequest.getDistance(), upStation, downStation);
+    }
+
+    public void update(LineRequest lineRequest) {
```

Comment:
중요한 것은 아닙니다만, 보통은 domain이 dto까지 알게 하지는 않고, (여기서는 단순 String 입니다만) VO를 각각 넣어주곤 합니다. layered architecture에서 dto는 보통 applicaion이나 ui 정도에 위치하기 때문에 더 안쪽 layer인 domain 에서 직접 사용하지는 않는 경향이 있는 듯 합니다 : )
```suggestion
    public void update(String name, String color) {
```

Reply:
그렇군요 domain 이 dto 까지 알게되면 dto의 변경이 domain 에 영향을 끼칠수도 있겠다 라는 생각이 듭니다..
각각 넣어주는 방법 이외에 dto < - > domain 사이에 서로를 연결하는 vo 를 두어도
둘 사이의 커플링을 끊을 수 있을것 같기도 한데 이 부분은 어떻게 생각하시나요??

Reply:
말씀하신 방법도 좋을 것 같습니다. 
`public void update(new Name(lineRequest.getName()), new Color(lineRequest.getColor()))`
대략 이런 형태가 될텐데, 지금 당장은 name이나 color에 제약 사항등이 많이 들어가지 않아 필요없어보이지만 서비스가 발전해나가면 VO로 분리하는 경우는 많이 있습니다!